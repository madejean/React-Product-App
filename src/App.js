import React from 'react';
import Products from './Products/Products.js';

class App extends React.Component {
  render() {
    return (
      <div>
        <header className="App-header">
          <h1>Products App!</h1>
        </header>
        <Products />
      </div>
    );
  }
}

export default App;
