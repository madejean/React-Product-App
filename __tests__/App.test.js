import React from 'react';
import ReactDOM from 'react-dom';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';
import App from '../src/App';

test('App component should render as expected', () => {
  const component = shallow(<App />)
  const tree = toJson(component)
  expect(tree).toMatchSnapshot()
})
