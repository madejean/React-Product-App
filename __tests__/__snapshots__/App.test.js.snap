exports[`test App component should render as expected 1`] = `
<div>
  <header
    className="App-header">
    <h1>
      Products App!
    </h1>
  </header>
  <Products />
</div>
`;
